client_response.go

```diff
@@ -27,6 +27,7 @@ type ClientResponse interface {
 	Code() int
 	Message() string
 	GetHeader(string) string
+	GetHeaders(string) string
```

Comment:
GetHeaders(string) []string ?

Reply:
Absolutely. Thanks for the quick review. I'll push the fix right away